frontend/src/app/components/action/Actions.css

```diff
@@ -37,3 +39,14 @@
   background: var(--surface-color-secondary-hover, #edebe9) !important;
   color: var(--typography-color-secondary, #474543) !important;
 }
+
+@media (max-width: 1198px) {
```

Comment:
For large screen bootstrap break i believe is 992px equals or greater , 1200 starts for extra large screens. Please use   bootstrap break as much possible.

Reply:
done